ql/cashflows/conundrumpricer.cpp

```diff
@@ -87,7 +95,7 @@ namespace QuantLib {
         fixingDate_ = coupon_->fixingDate();
         paymentDate_ = coupon_->date();
         const ext::shared_ptr<SwapIndex>& swapIndex = coupon_->swapIndex();
-        rateCurve_ = *(swapIndex->forwardingTermStructure());
+        rateCurve_ = *(swapIndex->discountingTermStructure());
```

Comment:
This should fall back to the forwarding term structure if the discounting term structure if null (hence the errors in the test).  Do you have time to do the change, or should I?  Thanks!